src/libraries/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.cs

```diff
@@ -102,24 +102,24 @@ public void SynchronizingObject_CalledOnEvent(WatcherChangeTypes expectedChangeT
                 if (expectedChangeType == WatcherChangeTypes.Created)
                 {
                     watcher.Created += dele;
-                    watcher.CallOnCreated(new FileSystemEventArgs(WatcherChangeTypes.Created, "test", "name"));
+                    watcher.CallNotifyFileSystemEventArgs(WatcherChangeTypes.Created, "name");
```

Comment:
These tests shouldn't be changing.  They were validating the behavior for when the OnXx methods were explicitly called, which, for example, a type derived from FileSystemWatcher could do (that's why these are all using a TestFileSystemWatcher, which is a derived type that just exposes those On methods for use).

So all of these tests changes should be reverted.  Instead, tests should be added that validate that actually making file changes which in turn trigger the FileSystemWatcher to invoke event handlers does so with those invocations in the right place.